models/uuv_hippocampus/uuv_hippocampus.sdf

```diff
@@ -0,0 +1,645 @@
+  <linkName>rotor_0</linkName>
+  <turningDirection>ccw</turningDirection>
+  <timeConstantUp>0.0125</timeConstantUp>
+  <timeConstantDown>0.025</timeConstantDown>
+  <maxRotVelocity>1100</maxRotVelocity>
+  <motorConstant>8.54858e-06</motorConstant>
+  <momentConstant>0.06</momentConstant>
+  <commandSubTopic>/gazebo/command/motor_speed</commandSubTopic>
+  <motorNumber>0</motorNumber>
+  <rotorDragCoefficient>0.000806428</rotorDragCoefficient>
+  <rollingMomentCoefficient>1e-06</rollingMomentCoefficient>
+  <motorSpeedPubTopic>/motor_speed/0</motorSpeedPubTopic>
+  <rotorVelocitySlowdownSim>0.05</rotorVelocitySlowdownSim>
+</plugin>
+<plugin name='back_left_motor_model' filename='libgazebo_uuv_motor_model.so'>
+  <robotNamespace/>
+  <jointName>rotor_1_joint</jointName>
+  <linkName>rotor_1</linkName>
+  <turningDirection>cw</turningDirection>
+  <timeConstantUp>0.0125</timeConstantUp>
+  <timeConstantDown>0.025</timeConstantDown>
+  <maxRotVelocity>1100</maxRotVelocity>
+  <motorConstant>8.54858e-06</motorConstant>
+  <momentConstant>0.06</momentConstant>
+  <commandSubTopic>/gazebo/command/motor_speed</commandSubTopic>
+  <motorNumber>0</motorNumber>
+  <rotorDragCoefficient>0.000806428</rotorDragCoefficient>
+  <rollingMomentCoefficient>1e-06</rollingMomentCoefficient>
+  <motorSpeedPubTopic>/motor_speed/0</motorSpeedPubTopic>
+  <rotorVelocitySlowdownSim>0.05</rotorVelocitySlowdownSim>
+</plugin>
+<plugin name='front_left_motor_model' filename='libgazebo_uuv_motor_model.so'>
+  <robotNamespace/>
+  <jointName>rotor_2_joint</jointName>
+  <linkName>rotor_2</linkName>
+  <turningDirection>ccw</turningDirection>
+  <timeConstantUp>0.0125</timeConstantUp>
+  <timeConstantDown>0.025</timeConstantDown>
+  <maxRotVelocity>1100</maxRotVelocity>
+  <motorConstant>8.54858e-06</motorConstant>
+  <momentConstant>0.06</momentConstant>
+  <commandSubTopic>/gazebo/command/motor_speed</commandSubTopic>
+  <motorNumber>0</motorNumber>
+  <rotorDragCoefficient>0.000806428</rotorDragCoefficient>
+  <rollingMomentCoefficient>1e-06</rollingMomentCoefficient>
+  <motorSpeedPubTopic>/motor_speed/0</motorSpeedPubTopic>
+  <rotorVelocitySlowdownSim>0.05</rotorVelocitySlowdownSim>
+</plugin>
+<plugin name='back_right_motor_model' filename='libgazebo_uuv_motor_model.so'>
+  <robotNamespace/>
+  <jointName>rotor_3_joint</jointName>
+  <linkName>rotor_3</linkName>
+  <turningDirection>cw</turningDirection>
+  <timeConstantUp>0.0125</timeConstantUp>
+  <timeConstantDown>0.025</timeConstantDown>
+  <maxRotVelocity>1100</maxRotVelocity>
+  <motorConstant>8.54858e-06</motorConstant>
+  <momentConstant>0.06</momentConstant>
+  <commandSubTopic>/gazebo/command/motor_speed</commandSubTopic>
+  <motorNumber>0</motorNumber>
+  <rotorDragCoefficient>0.000806428</rotorDragCoefficient>
+  <rollingMomentCoefficient>1e-06</rollingMomentCoefficient>
+  <motorSpeedPubTopic>/motor_speed/0</motorSpeedPubTopic>
+  <rotorVelocitySlowdownSim>0.05</rotorVelocitySlowdownSim>
+</plugin>-->
```

Comment:
Please remove the commented out parts

Reply:
those parts are now back in again and now spin the propellers/thruster